### PR TITLE
Removes whitespace issue that was preventing grunt init from working

### LINKED ISSUE #5454: Grunt init results in unterminated string constant error
### Issue Summary

Running `grunt init` in windows 8 with node version 0.12.4 results in an error and the command aborts.

Error:
```
Build failed.
File: ghost/templates/setup/three
Unterminated string constant (140:105)
SyntaxError: Unterminated string constant (140:105)
    at Parser.pp.raise (C:\Users\john\Code\Ghost\core\client\node_modules\ember-cli\node_modules\broccoli-es6modules\node_modules\esperanto\node_modules\acorn\dist\acorn.js:1745:13)
    ...
Warning: Command failed: C:\Windows\system32\cmd.exe /s /c "C:\Users\john\Code\Ghost\core\client\node_modules\.bin\ember build"
 Use --force to continue.

Aborted due to warnings.
```

### Steps to Reproduce

1. `npm install` (should pass successfully)
2. `grunt init`
3. See failure in all its glory

### Technical details:

* Ghost Version: master
* Client OS: Windows 8.1
* Node Version: 0.12.4

### PR DESCRIPTION
**Pull request by jxhn**
closes #5454
- removed newline and put the offending string on one line within the file

### CODE REVIEW

File: core/client/app/templates/setup/three.hbs [hunk 1]
```diff
@@ -8,8 +8,7 @@
 
     <form class="gh-flow-invite">
         <label>Enter one email address per line, we’ll handle the rest! <i class="icon-mail"></i></label>
-        {{textarea class="gh-input" name="users" placeholder="john@example.com
-sally.sanders@example.com" value=users}}
+        {{textarea class="gh-input" name="users" placeholder="john@example.com sally.sanders@example.com" value=users}}
```
Comment (novaugust) [inline core/client/app/templates/setup/three.hbs]: This placeholder is misleading; names are meant to be separated by newlines, not spaces. Better to just remove the `sally.sanders@example.com` part entirely

Comment (jomahoney) [inline core/client/app/templates/setup/three.hbs]: Yea that makes sense. I'll resubmit the PR.

Comment (novaugust) [inline core/client/app/templates/setup/three.hbs]: No need to resubmit, just commit --amend and force push :)

**Comment by novaugust** [comment]
I'm :+1: for this so that new windows contributors don't get explosions anymore. Looked around on stackoverflow, doesn't seem to be any standard way for having a multiline placeholder for textareas (in fact, the html spec says it should always be single line, but...).
Still, it's UX-y and so up to @JohnONolan

**Comment by JohnONolan** [comment]
Umm, no - sorry. Definitely not. The entire point of this placeholder is to show that you enter one email address per line. If we put them on one line we would be showing the user how to do the wrong thing.

**Comment by ErisDS** [comment]
Merged this as it's causing a build error. Raised a separate issue to look into how to fix this fully: #5459

**Pull request merged**